internal/host/pool_host_statemachine.go

```diff
@@ -99,6 +99,7 @@ func NewPoolHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler)
 	})
 
 	var hasMinRequiredHardware = stateswitch.And(If(HasMinValidDisks), If(HasMinCPUCores), If(HasMinMemory))
+	sufficientToBeMoved := stateswitch.And(hasMinRequiredHardware, If(IsNTPSynced))
```

Comment:
let's changed the name to sufficientToBeBinded

Reply:
OK - Just keep in mind that this term is already in  use in other places.

Reply:
Binded -> Bound